web/profiles/joinup/joinup.behat.inc

```diff
@@ -171,6 +174,53 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
   }
 
   /**
+   * Checks that the given select field has the given options.
+   *
+   * To do this we extract the text of the field which is the values of the
+   * select box concatenated. Then we check if the value is in this string.
```

Comment:
Rephrase it a bit. Something like:
> We extract the field text which contains the values expected to be found in the select box.
Then we check if all the options are available.